terraform/main.tf

```diff
@@ -92,6 +92,8 @@ resource "google_storage_bucket_iam_binding" "binding" {
   role   = "roles/storage.objectViewer"
   members = [
     "serviceAccount:rmi-beta-access@catalyst-cooperative-pudl.iam.gserviceaccount.com",
+    "serviceAccount:zerolab-beta-access@catalyst-cooperative-pudl.iam.gserviceaccount.com",
```

Comment:
It probably behooves us to eventually make a variable that holds all our beta users, so we don't have to duplicate that everywhere. But for now this seems OK.